### PR TITLE
feat(shared-liquidity): add default supply target utilization

### PR DESCRIPTION
**Pull request by Rubilmax**

### CODE REVIEW

File: packages/bundler-sdk-viem/src/operations.ts [hunk 1]
```diff
@@ -302,7 +310,8 @@ export const populateSubBundle = (
 
     const supplyTargetUtilization =
       publicAllocatorOptions.supplyTargetUtilization?.[market.params.id] ??
-      DEFAULT_SUPPLY_TARGET_UTILIZATION;
+      publicAllocatorOptions.defaultSupplyTargetUtilization ??
+      90_5000000000000000n;
```
Comment (oumar-fall) [inline packages/bundler-sdk-viem/src/operations.ts:314]: Why?

Comment (oumar-fall) [reply packages/bundler-sdk-viem/src/operations.ts:314]: I mean why not using it in a constant?

**Review by oumar-fall — DISMISSED** [review]
Why have you removed the constants?

File: packages/blue-sdk/src/constants.ts [hunk 2]
```diff
@@ -10,8 +10,6 @@ export const LIQUIDATION_CURSOR = 30_0000000000000000n;
  */
 export const MAX_LIQUIDATION_INCENTIVE_FACTOR = 1_150000000000000000n;
 
-export const ORACLE_PRICE_OFFSET = 36;
```
Comment (Rubilmax) [inline packages/blue-sdk/src/constants.ts:13]: This because I mis-used it in the API: it is badly named and is useless

**Review by oumar-fall — APPROVED** [review]

**Pull request merged**